backend/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "final-project",
+  "version": "1.0.0",
+  "description": "This is a final project for software engineering",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.20.1",
+    "butler-sdk": "0.0.2",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.1",
+    "mindee": "^2.0.0",
+    "moment-timezone": "^0.5.37",
+    "multer": "^1.4.5-lts.1",
+    "node-schedule": "^2.1.0",
+    "path": "^0.12.7"
+  },
+  "devDependencies": {
+    "nodemon": "^2.0.20"
+  }
+}
```

Comment:
I suggest that you could have remove unnecessary modules. Apart from that rest is good.